<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConductorLogFoo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/Icon_16.png</normaloff>../res/Icon_16.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="1" colspan="3">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Password :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="4">
         <widget class="QLineEdit" name="leIp">
          <property name="text">
           <string>10.101.50.60</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QLineEdit" name="lePass">
          <property name="text">
           <string>lights</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1" colspan="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>IP :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="4">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Connect to Database</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Username :</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="4">
         <widget class="QLineEdit" name="leUser">
          <property name="text">
           <string>etcuser</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="1" colspan="4">
         <widget class="QCommandLinkButton" name="btnConnect">
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="description">
           <string>Connect to the Conductor Database with the supplied credentials</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="0" colspan="3">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCommandLinkButton" name="btnGetLogs">
          <property name="text">
           <string>Get me logs</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Save To :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="leSavePath"/>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="btnSetPath">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lblResultInfo">
          <property name="text">
           <string>31276 records found</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTabWidget" name="timeSpecTab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Time Range</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>From</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="CalendarRangeWidget" name="cwFrom">
                 <property name="gridVisible">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="teFrom">
                 <property name="displayFormat">
                  <string>hh:mm:ss AP</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>To</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="CalendarRangeWidget" name="cwTo">
                 <property name="gridVisible">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="teTo">
                 <property name="displayFormat">
                  <string>hh:mm:ss AP</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Duration</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Retrieve logs for </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbDurationHours">
              <property name="minimum">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>hours</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbDurationMins">
              <property name="maximum">
               <number>59</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>minutes</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QRadioButton" name="rbBefore">
                <property name="text">
                 <string>Before</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbAfter">
                <property name="text">
                 <string>After</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbAround">
                <property name="text">
                 <string>Around</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="CalendarRangeWidget" name="cwDuration">
              <property name="gridVisible">
               <bool>true</bool>
              </property>
              <property name="verticalHeaderFormat">
               <enum>QCalendarWidget::ISOWeekNumbers</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <spacer name="verticalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QTimeEdit" name="teDuration">
                <property name="displayFormat">
                 <string>hh:mm:ss AP</string>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnJumpNext">
                <property name="text">
                 <string>Jump to next</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CalendarRangeWidget</class>
   <extends>QCalendarWidget</extends>
   <header>calendarrangewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leIp</tabstop>
  <tabstop>leUser</tabstop>
  <tabstop>lePass</tabstop>
  <tabstop>btnGetLogs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
